src/consumer/login/login.js

```diff
@@ -0,0 +1,83 @@
+import http from 'http';
+import uniqid from 'uniqid';
+import opn from 'opn';
+import os from 'os';
+import chalk from 'chalk';
+import { setSync, getSync } from '../../api/consumer/lib/global-config';
+import {
+  CFG_BITSRC_TOKEN_KEY,
+  CFG_USER_EMAIL_KEY,
+  DEFAULT_HUB_LOGIN,
+  CFG_HUB_LOGIN_KEY,
+  CFG_BITSRC_USERNAME_KEY
+} from '../../constants';
+import { LoginFailed } from '../exceptions';
+import logger from '../../logger/logger';
+
+const ERROR_RESPONSE = 500;
+const port = 8085;
```

Comment:
change to PORT

Reply:
Add the port as a flag, to bit login, and throw error explain to use it in case the port is in use.
otherwise user's using this port for something else won't be able to run the command